### DISCUSSION TITLE
Stream (live transcription) from a media player

### DISCUSSION THREAD

**Discussion by dgoryeo** [ggml-org/whisper.cpp#736] [Q&A]
By any chance is it possible to use the stream function to live transcribe/translate from a media player e.g. VLC?  Thanks.

**Comment by jihai** [comment]
Interesting idea! I think that's totally achievable. Here's my idea,

1. Create Multi-output channel from your mac
2. Use ./stream with -f to output the transcription to a file
3. Use a dummy bash script to retrieve the last line during runtime
4. (I don't know how media player works with srt) Find a way to inject the real time transcription
